app/cosmos/databases/queries.js

```diff
@@ -0,0 +1,25 @@
+const cosmosClient = require('../client')
+const { cosmosConfig } = require('../../config')
+
+const queriesDatabase = async () => {
+  try {
+    const { database } = await cosmosClient.databases.createIfNotExists({
+      id: cosmosConfig.queriesDatabase,
+      throughput: 400
+    })
+    await database.containers.createIfNotExists({
```

Comment:
Not something to change right now but an observation:

Having separate containers for queries and responses means we will have to write a query that can retrieve from both or we make two queries if we want to return an entire thread. Have you thought about which approach might be best?

Reply:
We can look at how that works as part of the follow on ticket for the frontend